classes/app.rb

```diff
@@ -65,6 +72,14 @@ def option(input)
     # rubocop:enable Metrics
   end
 
+  def exit_app
+    save_music_album
+    save_genre
+    puts
+    puts 'Thank You for using my myCatalog Library!'
+    puts 'Built with 💖 by Bright | Dejan | Alick 🗽'
+  end
```

Comment:
This method doesn't save data on exit. Kindly fix it.

Reply:
Done!